src/index.ts

```diff
@@ -31,5 +31,8 @@ export const attach = (element: HTMLTextAreaElement) => {
     detach() {
       element.removeEventListener("input", inputHandler);
     },
+    update() {
+      inputHandler();
+    },
```

Comment:
```suggestion
    update: inputHandler,
```

Reply:
But I would actually think about adding something like that directly to the attach function:
```ts
const elementPrototype = Object.getPrototypeOf(element);
const descriptor = Object.getOwnPropertyDescriptor(elementPrototype, 'value');
Object.defineProperty(element, 'value', {
    ...descriptor,
    set() {
        descriptor.set.apply(this, arguments);
        inputHandler();
    },
});
```

So as you said - observer in this library code. But IMO it's better to have it handled once here than having to do it separately in every library out there.

Reply:
so maybe, both?

Reply:
Yea, having an update method exposed to the public is cool imo